pages/TheExplorationLibrary.html

```diff
@@ -5,28 +5,26 @@
        aria-hidden="true">
       <span class="octicon octicon-link"></span>
     </a>
-    The Exploration Gallery
+    The Exploration Library
   </h2>
 
   <p>
     After signing in to Oppia, you'll end up on the Home page. Even before you
     start creating your own explorations, there are a number of things you can
-    do. You can click on the icon at the top left corner of the page to access
-    the blog, join the discussion forum or access the Participation Playbook.
+    do. Hover the cursor over the about menu in the navigation bar at the top right of the page to access the blog, join the discussion forum or learn how to get started.
```

Comment:
Consider keeping lines to 80 chars or so, similar to the rest of the file -- it makes it a bit easier to read when reviewing.

Also, suggest: "about menu" --> "About menu"